stock_warehouse_calendar/tests/test_stock_warehouse_calendar.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2018-19 ForgeFlow S.L. (https://www.forgeflow.com)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import fields
+from odoo.tests.common import TransactionCase
+
+
+class TestStockWarehouseCalendar(TransactionCase):
+    def setUp(self):
+        super(TestStockWarehouseCalendar, self).setUp()
```

Comment:
Use short notation:

```suggestion
        super().setUp()
```

Reply:
Done, also disabled tracking in the tests, always worth it.